src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/geo/GEOMiner.java

```diff
@@ -217,21 +216,18 @@ public void onBlockBreak(@Nonnull Block b) {
         };
     }
 
-    @Nonnull
     @Override
-    public int[] getInputSlots() {
+    public @Nonnull int[] getInputSlots() {
```

Comment:
Null isn't passable on primitive but ig maybe it is for primitive arrays? Not sure but either way this is.... weird

Reply:
the compiler lets you return null for an int array if you want.